packages/experiments/src/components/SelectedItemsList/examples/SelectedPeopleList.Basic.Example.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from 'office-ui-fabric-react/lib/Utilities';
 import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { people, groupOne, groupTwo } from 'experiments/lib/ExtendedPicker';
-import '../../../../../office-ui-fabric-react/src/components/Pickers/PeoplePicker/examples/PeoplePicker.Types.Example.scss';
+import 'office-ui-fabric-react/src/components/Pickers/PeoplePicker/examples/PeoplePicker.Types.Example.scss';
```

Comment:
I think this should be /lib/

Reply:
the src is not included in the npm published package and is not going to be accessible.